kaptan/__init__.py

```diff
@@ -125,6 +125,56 @@ def get(self, key=None, default=SENTINEL):
                 return default
             raise
 
+    def add(self, key=None, value=None, replace=False):
```

Comment:
Why `key` field may be null ?

I think, you should remove the following code

```python
if not key:
    raise RuntimeError("Unable to find key")
```

and `key` field must be mandatory.